config/api.yml

```diff
@@ -88,6 +88,34 @@
        :post:
        - :name: create
          :identifier: alert_status_action_new
+  :alert_definitions:
+    :description: Alert Definitions
+    :identifier: alert_definition
+    :options:
+    - :collection
+    :verbs: *gpd
+    :klass: MiqAlert
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: alert_definitions_show_list
+      :post:
+      - :name: create
+        :identifier: alert_definition_new
+      - :name: delete
```

Comment:
@dkorn looks like we're missing bulk edits,  i.e. POST action "edit" on the collection.  When you get a chance, can you create an enhancement PR for that and add the appropriate test ? Thanks.